libs/proxy/tokio-postgres2/src/error/sqlstate.rs

```diff
@@ -1,1670 +1,817 @@
-// Autogenerated file - DO NOT EDIT
```

Comment:
I need to figure out the future of maintaining this file. I think many of the consts can disappear without worry.

Reply:
removed all the auto-generated consts